neo4j/manifest.json

```diff
@@ -16,7 +16,7 @@
   ],
   "public_title": "Datadog-Neo4j Integration",
   "categories": [
-    "database"
+    "data-store"
```

Comment:
Shouldnt it be "data store"?

Reply:
data-store category should be correct. the integrations page lists it as DATA-STORE: https://a.cl.ly/2Nuz8GGP
That said, this has to be rebuilt and looked at. This PR is a recommendation to proceed. Let me know if I should ping somebody or follow up on this.